docs/chain_documentation/classsifications/classifications.md

```diff
@@ -1,6 +1,15 @@
 # Classifications
 
+
+
+![banner_urban_typo](../../resources/images/chain_documentation/banner_urban_typo.png)
+
+
+
 On the basis of the previously calculated [indicators](../indicators/indicators.md), the Geoclimate chain allows the territory to be segmented using the following two classifications:
```

Comment:
There is some confusion in the text between geoclimate chain and geoclimate algorithms (processes).
Classification is only related to a process that use indicators.

Reply:
So what about 
`On the basis of the previously calculated indicators, we propose two classifications to segment the territory`